boot/core/src/boot/task_helpers.clj

```diff
@@ -35,21 +35,11 @@
         (->> sym ns-aliases (into base) (mapcat pubs)))
       (filter task?) (sort-by :name) (group-by :ns*) (into (sorted-map)))))
 
-(defn read-pass
-  [prompt]
-  (String/valueOf (.readPassword (System/console) prompt nil)))
-
```

Comment:
@micha it's still in `boot.pgp` but I agree that a fn reading user input could be in `boot.util` as well.